### DISCUSSION TITLE
Can I install a Harman AIR .apk file on Fire TV Stick 4k?

### DISCUSSION THREAD

**Discussion by as3wizard** [airsdk/Adobe-Runtime-Support#1637] [General]
Has anyone ever installed Harman or Adobe AIR .apk file on Fire TV Stick 4k?

I received this error:
"There was a problem parsing the package"

I used the amazon firestick app "Downloader" to get the apk file on the stick by downloading it from a website called ufile dot io. I created the .apk file using Adobe Animate 2021 & the latest Harmon AIR SDK compiler

I'll keep trying things & update this thread if I figure it out. Please let me know if it's not possible.
Thanks

**Comment by lwmirkk** [comment]
Yes, you can install e download with DOWNLOADER, but you can connect the Fire Stick via USB on PC and install with ADB (enable USB debugging in Fire Stick before in Developer mode).

commands:
(to check if the device is shown).
`adb devices`

(probrably you will be prompted in Fire Stick to enable the PC to connect to Fire Stick, accept it (and mark the remember button)

(open the CMD in the same folder you have the APK to be easy)
`adb install APKFILEPATH.apk `

or

`adb -s DEVICE_ID -r install APKfilename.apk`

-s = to target this device if more than one are connected to PC
-r = to replace the current app/apk you installed previously (if the app is already installed in the fire stick)..

Fire Stick is hard to install APK without ADB. Xiaomi Mi Stick (or Mi Box) or any other with Android/Google TV and Play Store will be easier.
